iOSClient/Networking/NCNetworking.swift

```diff
@@ -173,8 +173,10 @@ import Queuer
             
             var secresult = SecTrustResultType.invalid
             let status = SecTrustEvaluate(serverTrust, &secresult)
-            if errSecSuccess == status, let serverCertificate = SecTrustGetCertificateAtIndex(serverTrust, 0) {
-                    
+            let isServerTrusted = SecTrustEvaluateWithError(serverTrust, nil)
+
+            if let serverCertificate = SecTrustGetCertificateAtIndex(serverTrust, 0) {
```

Comment:
Can you combine the two `if`s? And then then use `serverCertificate` with `saveX509Certificate(serverCertificate, [...])` directly?